src/main/java/org/cascadebot/cascadebot/commands/music/QueueMoveSubCommand.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2020 CascadeBot. All rights reserved.
+ * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.commands.music;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Member;
+import org.cascadebot.cascadebot.UnicodeConstants;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.commandmeta.ISubCommand;
+import org.cascadebot.cascadebot.messaging.MessageType;
+import org.cascadebot.cascadebot.messaging.MessagingObjects;
+import org.cascadebot.cascadebot.music.MovableAudioTrack;
+import org.cascadebot.cascadebot.permissions.CascadePermission;
+import org.cascadebot.cascadebot.utils.buttons.Button;
+import org.cascadebot.cascadebot.utils.buttons.ButtonGroup;
+import org.cascadebot.cascadebot.utils.move.MovableList;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class QueueMoveSubCommand implements ISubCommand {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        if (context.getArgs().length >= 2) {
```

Comment:
This command only accepts two arguments or no arguments right? If so, we should send a usage if they enter >2 or 1 argument

Reply:
No it accepts 1 argument as a start position for using the gui move, but I forgot to implement that.

Reply:
Ah okay, anyway, we should still throw a usage if it's > 2 length